js/apps/events-page/services/dateUtilityService.js

```diff
@@ -0,0 +1,68 @@
+((app) => {
+	const dateUtilityService = () => {
+		const addDays = (dateOrString, daysToAdd) => {
+			const date = typeof dateOrString === 'string' ? new Date(dateOrString) : dateOrString;
+
+			if (typeof date !== 'object' || date === 'Invalid Date') {
+				return date;
+			}
+
+			date.setDate(date.getDate() + daysToAdd);
+
+			return date;
+		};
+
+		const formatSchedule = (eventStart, eventLength) => {
+			if (!eventStart || isNaN(Date.parse(eventStart))) {
+				return 'Bad start date format';
+			}
+
+			if (typeof eventLength !== 'number' || eventLength <= 0) {
+				return 'Bad event length format';
+			}
+
+			const eventStartDate = new Date(eventStart);
+			const eventEndDate = new Date(eventStart);
+			eventEndDate.setMinutes(eventStartDate.getMinutes() + eventLength);
+
+			const startHour = get12HourValue(eventStartDate);
+			const startMinutes = getMinuteString(eventStartDate.getMinutes());
+			const startAmPm = getAmPm(eventStartDate);
+			const endHour = get12HourValue(eventEndDate);
+			const endMinutes = getMinuteString(eventEndDate.getMinutes());
+			const endAmPm = getAmPm(eventEndDate);
+
+			return `${startHour}:${startMinutes} ${startAmPm} to ${endHour}:${endMinutes} ${endAmPm}`;
+		};
+
+		const get12HourValue = (date) => {
+			const rawHours = date.getHours();
+
+			if (rawHours === 0) return 12;
```

Comment:
did this pass lint without the brackets?

Reply:
Yes, and it's also mentioned in the Airnbnb JS standards: https://github.com/airbnb/javascript#blocks--braces